src/oas2/accessors.ts

```diff
@@ -43,5 +43,5 @@ function getSecurity(
 
 function getProducesOrConsumes(which: 'produces' | 'consumes', spec: Partial<Spec>, operation: Partial<Operation>) {
   const mimeTypes = get(operation, which, get(spec, which, [])) as string[];
-  return mimeTypes.length ? mimeTypes : ['*/*'];
```

Comment:
@XVincentX, why do we need this? Is this consistent with OAS3?

Reply:
@lag-of-death This is the inconsistency you found while working on the PR yesterday; empty content in OAS3, but */* for OAS2 — which is technically incorrect. Does that clarify?

Reply:
@XVincentX, I'm sorry, I must have been colorblind there for a sec.. all good, my bad!